python/ray/_raylet.pyx

```diff
@@ -262,23 +262,31 @@ class ObjectRefGenerator:
 
 
 class StreamingObjectRefGenerator:
```

Comment:
not blocking this PR. Should we move this class to a different file? this file is pretty long. also, is there a perf improvement to define it in Cython? It seems to me that most code in this class is pure Python code.

Reply:
I think we can definitely move out of it. I will handle it in the second PR together